webapp/src/components/PageHeader.tsx

```diff
@@ -186,10 +193,34 @@ const PageHeader = () => {
                 },
               }}
             >
-              <Settings />
+              <SettingsIcon />
             </IconButton>
             <HelpMenu />
           </Box>
+          <Dialog
+            aria-labelledby="config-dialog-title"
+            maxWidth="md"
+            scroll="paper"
+            open={openConfigModal}
+            onClose={(_, reason) =>
+              reason === "backdropClick" || reason === "escapeKeyDown"
+                ? setOpenConfigModal(true)
+                : setOpenConfigModal(false)
+            }
```

Comment:
Following up on [this](https://github.com/ServiceNow/azimuth/pull/437#discussion_r1124014825).
You also inverted the `true` and `false`, so the dialog box doesn't close now.
I insist on simplifying this to
```suggestion
            onClose={() => setOpenConfigModal(false)}
```

Reply:
Okay, we need to stay on the modal for both reasons (backdrop click and esckeydown) when there are ongoing changes or updates. Users may accidentally do backdrop click or esc.

Reply:
If that's what you want, you should simply not define `onClose`.
```suggestion
```

Reply:
I can use "disableEscapeKeyDown" to stop calling on close if the ESC key is pressed. But, I need to define inside onClose to control the backdrop click. Would you happen to have any other suggestions to handle this? Do you feel this check is optional?

Reply:
Thanks. I did not define onClose and it handled both reasons with ease.